doc/examples/features_detection/plot_haar.py

```diff
@@ -0,0 +1,70 @@
+"""
+============================
+Haar-like feature descriptor
+============================
+
+Haar-like feature descriptors are a set of simple features which was used in
+the first real-time face detector [1]_. These features can be efficiently
+computed on any scale in constant time, using an integral image [1]_ and by
```

Comment:
make it simpler:
 These features can be efficiently
 computed on any scale in constant time, using an integral image [1]_.